saga/saga.go

```diff
@@ -199,21 +205,32 @@ func (s *Saga) updateSagaState(msgs []SagaMessage) error {
 // updated in a thread safe manner
 // Processes at most SagaUpdateChSize number of updates at a time
 func (s *Saga) updateSagaStateLoop() {
+	var loopLatency = s.stat.Latency(stats.SagaUpdateStateLoopLatency_ms)
```

Comment:
you can append .Time() here and ditch the additional line below

Reply:
yup!